examples/projections/conic/conic_albers.py

```diff
@@ -1,9 +1,20 @@
 """
 Albers Conic Equal Area
 =======================
+This projection, developed by Heinrich C. Albers in 1805, is predominantly used to map
```

Comment:
I think the source codes look better if we add a blank space between the section title and the paragraph, but it's just my personal taste.

```suggestion
=======================

This projection, developed by Heinrich C. Albers in 1805, is predominantly used to map
```

Reply:
Just pushed a change with a blank line added to all of the docstrings